tools/create_operator.py

```diff
@@ -0,0 +1,318 @@
+        "a",
+        encoding="utf-8",
+    ) as file:
+        file.write(
+            f"""
+py_library(
+    name = "{lower_op}",
+    srcs = ["{lower_op}.py"],
+    srcs_version = "PY3",
+    deps = [
+        ":base",
+        "//temporian/core:operator_lib",
+        "//temporian/core/data:event",
+        "//temporian/core/data:feature",
+        "//temporian/proto:core_py_proto",
+    ],
+)
+    """
+        )
+
+    # Operator implementation
+    with open(
+        os.path.join(
+            "temporian",
+            "implementation",
+            "numpy",
+            "operators",
+            lower_op + ".py",
+        ),
+        "w",
+        encoding="utf-8",
+    ) as file:
+        file.write(
+            f'''{licence()}
+
+"""Implementation for the {capitalized_op} operator."""
+
+
+from typing import Dict
+
+from temporian.implementation.numpy.data.event import NumpyEvent
+from temporian.core.operators.{lower_op} import {capitalized_op}
+from temporian.implementation.numpy import implementation_lib
+from temporian.implementation.numpy.operators.base import OperatorImplementation
+
+class {capitalized_op}NumpyImplementation(OperatorImplementation):
+
+    def __init__(self, operator: {capitalized_op}) -> None:
+        assert isinstance(operator, {capitalized_op})
+        super().__init__(operator)
+
+    def __call__(
+        self, event: NumpyEvent) -> Dict[str, NumpyEvent]:
+
+        return {{"event": event}}
+
+
+implementation_lib.register_operator_implementation(
+    {capitalized_op}, {capitalized_op}NumpyImplementation
+)
+'''
+        )
+
+    # Operator implementation build
+    with open(
+        os.path.join(
+            "temporian", "implementation", "numpy", "operators", "BUILD"
+        ),
+        "a",
+        encoding="utf-8",
+    ) as file:
+        file.write(
+            f"""
+py_library(
+    name = "{lower_op}",
+    srcs = ["{lower_op}.py"],
+    srcs_version = "PY3",
+    deps = [ ":base",
+        "//temporian/core/data:duration",
+        "//temporian/core/operators:{lower_op}",
+        "//temporian/implementation/numpy:implementation_lib",
+        "//temporian/implementation/numpy:utils",
+        "//temporian/implementation/numpy/data:event",
+        "//temporian/implementation/numpy/data:sampling",
+    ],
+)
+
+    """
+        )
+
+    # Operator implementation test
+    with open(
+        os.path.join(
+            "temporian",
+            "implementation",
+            "numpy",
+            "operators",
+            "test",
+            lower_op + "_test.py",
+        ),
+        "w",
+        encoding="utf-8",
+    ) as file:
+        file.write(
+            f"""{licence()}
+
+from absl.testing import absltest
+
+import pandas as pd
+from temporian.core.operators.{lower_op} import {capitalized_op}
+from temporian.implementation.numpy.operators.{lower_op} import (
+    {capitalized_op}NumpyImplementation,
+)
+from temporian.implementation.numpy.data.event import NumpyEvent
```

Comment:
nit: move this import from line 243 to line 240 (sort imports in alphabetical order)

Reply:
*Done